Source/Scene/Cesium3DTileset.js

```diff
@@ -344,22 +354,25 @@ define([
         if (!outOfCore) {
             return;
         }
-        if (!hasAvailableRequests(tiles3D)) {
+        if (!hasAvailableRequests(tile)) {
             return;
         }
 
-        var stats = tiles3D._statistics;
-        ++stats.numberOfPendingRequests;
-        addLoadProgressEvent(tiles3D);
-
         tile.requestContent();
-        var removeFunction = removeFromProcessingQueue(tiles3D, tile);
-        when(tile.processingPromise).then(addToProcessingQueue(tiles3D, tile)).otherwise(removeFunction);
-        when(tile.readyPromise).then(removeFunction).otherwise(removeFunction);
+
+        if (!tile.isContentUnloaded()) {
+            var stats = tiles3D._statistics;
+            ++stats.numberOfPendingRequests;
+            addLoadProgressEvent(tiles3D);
+
+            var removeFunction = removeFromProcessingQueue(tiles3D, tile);
+            when(tile.processingPromise).then(addToProcessingQueue(tiles3D, tile)).otherwise(removeFunction);
+            when(tile.readyPromise).then(removeFunction).otherwise(removeFunction);
+        }
     }
 
-    function hasAvailableRequests(tiles3D) {
-        return RequestScheduler.hasAvailableRequests(tiles3D._url);
+    function hasAvailableRequests(tile) {
+        return !defined(tile._requestServer) || tile._requestServer.hasAvailableRequests();
```

Comment:
It's not defined when the tile is empty, but I suppose also with data URI when we have them.